marianas_process_loop_umi_1.8.0/container/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM java:8
```

Comment:
Also, we can use `openjdk:8`, but it's ok just FYI.

Reply:
will change this